compiler/src/iree/compiler/InputConversion/StableHLO/Preprocessing/Canonicalization.cpp

```diff
@@ -1087,5 +1123,29 @@ void populateCanonicalizationPatterns(MLIRContext *context,
       ReshapeOpCanon, TransposeOpCanon,
       // Types.
       ZeroExtentTensorCanon>(context, benefit);
+  patterns
+      ->add<ReorderElementwiseAndShapeOp<mlir::stablehlo::AbsOp>,
```

Comment:
No changes needed: it would be nice to just have the list of Ops in a helper and make it part of the matcher. I don't think its actually that much better though.

Reply:
Added a matcher for the elemenetwise trait + check for one input as @jpienaar suggested. Seems like the cleanest approach overall